lib/order.rb

```diff
@@ -8,11 +13,60 @@ def initialize(id, products)
     end
 
     def total
-      # TODO: implement total
+      sum = 0
+      @products.values.each do |price|
+        sum += price
+      end
+      cents_with_tax = (sum + (sum * 0.075))*100.round
+      sum_with_tax = Money.new(cents_with_tax, "USD")
```

Comment:
1.  It's neat that you're using the "Money" gem.
1.  I don't think it's needed here.